internal/services/springcloud/spring_cloud_build_deployment_resource.go

```diff
@@ -56,6 +56,20 @@ func resourceSpringCloudBuildDeployment() *pluginsdk.Resource {
 				ValidateFunc: validation.StringIsNotEmpty,
 			},
 
+			"active": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Computed: true,
+			},
+
+			"addon_json": {
+				Type:             pluginsdk.TypeString,
+				Optional:         true,
+				Computed:         true,
```

Comment:
why is this `Computed: true`? Does the API return a default value here?

Reply:
Yes